assignment.rb

```diff
@@ -0,0 +1,149 @@
+# 2.2 Super Classes - Andrew T. Poe
+# Dinosaurs
+
+class Dinosaur
+  attr_accessor :is_alive, :number_of_legs, :habitat, :size, :fight_level,
+                         :defense_level, :mode_of_transportation, :is_mother
+
+  def initialize
+    @is_alive = true
+  end
+
+  def fight(victim)
+    defense_level = victim.defense_level
```

Comment:
since `victim.defense_level` isn't used in more than 1 spot, i'd put it on line 14 on the other side of `>`